ddp-workspace/projects/ddp-dsm-ui/src/app/activity-data/components/matrix-answer-table.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, Input} from '@angular/core';
+import {QuestionDefinition, Row} from '../models/question-definition.model';
+import {MatrixAnswer} from '../models/question-type-models';
+import {Option} from '../models/option.model';
+
+@Component({
+  selector: 'app-matrix-answer-table',
+  templateUrl: './matrix-answer.table.component.html',
+  styleUrls: ['./matrix-answer.table.component.scss'],
+})
+export class MatrixAnswerTableComponent {
+  @Input() answers: MatrixAnswer[];
+  @Input() questionDefinition!: QuestionDefinition;
+
+  public isChecked(optionId: string, rowId: string): boolean {
+    const checkedAnswerIndex = this.answers
+      .findIndex(answer => answer.horizontalAnswer?.optionStableId === optionId && answer.verticalAnswer?.rowStableId === rowId);
+    return checkedAnswerIndex > -1;
```

Comment:
Hi Giorgi,
you can use `Array.prototype.some` for that purpose.
It returns boolean value.